.azure-pipelines/templates/jobs/snap-deploy-job.yml

```diff
@@ -30,7 +29,7 @@ jobs:
   # https://docs.microsoft.com/en-us/azure/devops/pipelines/library/secure-files?view=azure-devops#q-how-do-i-authorize-a-secure-file-for-use-in-a-specific-pipeline.
   #
   # This file has a maximum lifetime of one year and the current file will
-  # expire on 2023-09-06. The file will need to be updated before then to
+  # expire on 2024-02-10. The file will need to be updated before then to
```

Comment:
do we need to re-authorize this once merged?

Reply:
Do you mean authorizing the new credentials for the "nightly" and "release" pipelines as described above?

If so, nope! I went ahead and updated the credentials last night. I had to reauthorize things as a part of that, but that's all been done.